protostar/commands/test/test_command.py

```diff
@@ -188,4 +197,19 @@ async def test(
                     exit_first=exit_first,
                 )
 
+            try:
+                if (
+                    report_slowest_tests_count
+                    and (len(testing_summary.failed) + len(testing_summary.passed)) > 0
+                ):
+                    logger.info(log_color_provider.bold("Slowest test cases:"))
+                    print(
+                        get_formatted_slow_tests(
+                            testing_summary, report_slowest_tests_count
+                        ),
+                        end="\n\n",
+                    )
```

Comment:
We need to keep `TestCommand` from growing. If you move the logic to the `TestingSummary` you can pass slowest_tests_count to that class and move this logic there.